### PR TITLE
added file with acronyms

### PR DESCRIPTION
**Pull request by VictorSyntez**
Following Issue#2 I've created file with acronyms. Feel free to add content there.

### CODE REVIEW

**Review by vsnt — APPROVED** [review]

File: Acronyms.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+# Acronyms
+
+APAC = Asia Pacific
+CCG = Credentials Community Group
+DHS = Department of Homeland Security
+DIACC = Digital Identity & Authentication Council of Canada
+DID = Decentralized Identifier
+DIF - Decentralized Identity Foundation
```
Comment (TallTed) [inline Acronyms.md:12]: duplicate line (aside from typo of hyphen replacing equal-sign)
```suggestion
```

File: Acronyms.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+DIF = Decentralized Identity Foundation
+IPR = Intellectual Property Rights
+SVIP = Silicon Valley Innovation Program (at DHS)
+TF - Task Force
```
Comment (TallTed) [inline Acronyms.md:16]:
```suggestion
TF = Task Force
```

File: Acronyms.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ToIP = Trust over IP (Linux Foundation Group)
+VC = Verifiable Credential
+W3C = World Wide Web Consortium
+WG - Working Group
```
Comment (TallTed) [inline Acronyms.md:20]:
```suggestion
WG = Working Group
```

**Comment by TallTed** [comment]
Just realized, already merged...  Merged too quickly!

**Pull request merged**